src/components/map.js

```diff
@@ -1,30 +1,57 @@
 import React, { useRef, useEffect, useState } from "react";
 import mapboxgl from "mapbox-gl";
+import db from "../util/firebase";
+import ReactMapGL, { Marker, Popup } from "react-map-gl";
 
 mapboxgl.accessToken = `${process.env.REACT_APP_MAPBOX_KEY}`;
 
 const Map = () => {
+  //map config
   const mapContainer = useRef();
   const [lng, setLng] = useState(4.9041);
   const [lat, setLat] = useState(52.3676);
   const [zoom, setZoom] = useState(12);
 
-  useEffect(() => {
+  //chains data
+  const [chainData, setChainData] = useState([]);
+
+  const getData = async () => {
+    return db
+      .collection("chains")
+      .get()
+      .then(async (snapshot) => {
```

Comment:
You can use the `getChain` function for this from `util/chain`

Reply:
Going to update this and recommit